package.json

```diff
@@ -14,6 +14,7 @@
     "sake": "github:skyverge/sake"
   },
   "scripts": {
-    "generate:category_attribute_json": "php bin/GenerateCategoryAttributeMapping.php"
+    "generate:category_attribute_json": "php bin/GenerateCategoryAttributeMapping.php",
+    "lint:php": "vendor/bin/phpcs . -p -s --colors "
```

Comment:
> Can you clarify the intended use of this script

I just added this to the readme in 6608a82cfa41ad5190ff4fe572312f94f57d2387

My recommendation would be to this before pushing to Github. Ideally this would only display errors that you have just introduced but at the moment it's a bit less useful because of all the PHPCS violations.

> should our actions use the same script?

No, they should be formatted differently.

Reply:
> display errors that you have just introduced

I think I've seen projects use a git command to check only changed files, and/or hook that up as a pre-commit action. Maybe a follow up for next cooldown :)

Reply:
That would be cool!